### PR TITLE
Subquery indexer

### PR DESCRIPTION
**Pull request by ba1ciu**

### CODE REVIEW

File: indexer/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT LICENSE
+
+Copyright 2020-2021 OnFinality Limited authors & contributors
```
Comment (gjermundgaraba) [inline indexer/LICENSE:3]: Is this license necessary? The code we have here is ours, or no?

File: indexer/README.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
+# SubQuery - Starter Package for Cosmos/Stargaze
```
Comment (gjermundgaraba) [inline indexer/README.md:1]: Readme correct here?

File: indexer/LICENSE (hunk 1 above)
Comment (ba1ciu) [reply indexer/LICENSE]: Yep, it is. It's a leftover from a starter project. Removed

File: indexer/README.md (hunk 2 above)
Comment (ba1ciu) [reply indexer/README.md]: Not really :) I fixed it.

**Comment by ba1ciu** [comment]
@gjermundgaraba https://github.com/EmpowerPlastic/empowerchain/pull/805/commits/d75d22dad45c7318dcffcd149e8a91a7f1554a8a

**Review by gjermundgaraba — APPROVED** [review]

**Pull request merged**